### PR TITLE
Added initial expansion pack logic #139

### PR DESCRIPTION
**Pull request by DreyerChris**

### CODE REVIEW

**Comment by DreyerChris** [comment]
Closes #139

File: src/components/Lobby/index.tsx [hunk 1]
```diff
@@ -46,13 +48,14 @@ const LobbyScreen = ({ onStartGame }: LobbyScreenProps) => {
 
   const handleStartGame = () => {
     const selectedNames = playerNames.slice(0, playerCount).map((name, index) => name || `Player ${index + 1}`);
+    console.log(useExpansionPack);
```
Comment (H1D) [inline src/components/Lobby/index.tsx:51]:
```suggestion
    
```

File: src/decks/ecosfera-baltica.deck.json [hunk 2]
```diff
@@ -82,6 +82,62 @@
...
+      "theme": "climateChange",
+      "description": "If min. one measure has been implemented when this card is lifted, all players can activate all action-tokens. If no measures have been implemented all players need to deactivate all action-token.",
+      "usage": "single"
+    },
+    "Climate change": {
+      "effect": "negative",
+      "theme": "climateChange",
+      "description": "Each time a player gets an impact-tile an additional impact-tile is added to the impact tile puzzle.",
+      "usage": "permanent"
+    }
```
Comment (H1D) [inline src/decks/ecosfera-baltica.deck.json:139]: We will also need a title for card ("Focus" column from the spreadsheet)

File: src/state/types.ts [hunk 3]
```diff
@@ -95,7 +104,10 @@ export type StageEventType =
   | "habitatUnlock"
   | "cardBuy"
   | "gameLoss"
-  | "gameWin";
+  | "gameWin"
+  | "policy_specialDraw"
+  | "policy_fundingIncrease"
+  | "policy_climateChange";
```
Comment (H1D) [inline src/state/types.ts:110]: Lets invert that. Those types should come from @/state/machines/expansion/index.ts

```ts
| ExtPackEvent;
```

File: src/decks/ecosfera-baltica.deck.json (hunk 2 above)
Comment (DreyerChris) [reply src/decks/ecosfera-baltica.deck.json:139]: Yeah this is the name of the card (the key of this object)

**Review by H1D — APPROVED** [review]

**Pull request merged**